Content.Client/GameObjects/EntitySystems/AmbienceSystem.cs

```diff
@@ -90,6 +134,35 @@ private void EndAmbience()
             _ambientStream.Stop();
             _ambientStream = null;
         }
+
+        private void LobbyMusicCVarChanged(bool musicEnabled)
+        {
+            if (!musicEnabled)
+            {
+                EndLobbyMusic();
+            }
+            else if (_stateManager.CurrentState is LobbyState)
```

Comment:
You forgot the case where musicEnabled && !_stateManager.CurrentState is LobbyState - might want to call EndLobbyMusic() in that case just to be safe